### DISCUSSION TITLE
Is this repository alive ?

### DISCUSSION THREAD

**Discussion by vault-thirteen** [ProtonMail/go-crypto#160] [Q&A]
Hello there.

This repository is used by "github.com/go-git/go-git/v5" and its dependency brought me here.
And here I see no tags and no releases.
Why is it so empty ?
Is this repository alive ?

Thank you.

**Comment by twiss** [comment]
Hi @vault-thirteen, yes, this repo is alive and under active development, but we indeed don't tag releases at the moment. For most things we try to keep backwards compatibility. We may start to tag releases in the future, though.
